src/dotPlotSettingsModel.ts

```diff
@@ -0,0 +1,201 @@
+            displayNameKey: "Visual_TextSize",
+            value: dataLabelUtils.DefaultFontSizeInPt,
+            options: {
+                minValue: { value: 8, type: ValidatorType.Min },
+                maxValue: { value: 60, type: ValidatorType.Max },
+            }
+        }),
+        fontFamily: new formattingSettings.FontPicker({
+            name: "fontFamily",
+            value: "Arial, sans-serif"
+        }),
+        bold: new formattingSettings.ToggleSwitch({
+            name: "bold",
+            value: false,
+        }),
+        italic: new formattingSettings.ToggleSwitch({
+            name: "italic",
+            value: false,
+        }),
+        underline: new formattingSettings.ToggleSwitch({
+            name: "underline",
+            value: false,
+        }),
+    });
+
+
+    labelDisplayUnits = new formattingSettings.AutoDropdown({
+        name: "labelDisplayUnits",
+        displayName: "Display units",
+        displayNameKey: "Visual_DisplayUnits",
+        description: "Select the units (millions, billions, etc.)",
+        descriptionKey: "Visual_Description_DisplayUnits",
+        value: 0,
+    });
+
+    labelPrecision = new formattingSettings.NumUpDown({
+        name: "labelPrecision",
+        displayName: "Decimal places",
+        displayNameKey: "Visual_DecimalPlaces",
+        description: "Select the number of decimal places to display",
+        descriptionKey: "Visual_Description_DecimalPlaces",
+        value: 2,
+        options: {
+            minValue: { value: 0, type: ValidatorType.Min },
+            maxValue: { value: 17, type: ValidatorType.Max },
+        }
+    });
+
+    orientation = new formattingSettings.ItemDropdown({
+        name: "orientation",
+        displayName: "Orientation",
+        displayNameKey: "Visual_Orientation",
+        items: orientationOptions,
+        value: orientationOptions[0],
+    });
+
+    topLevelSlice = this.show;
+    name = "labels";
+    displayName = "Data labels";
+    displayNameKey = "Visual_DataLabels";
+    description = "Display data label options";
+    descriptionKey = "Visual_Description_DataLabels";
+    slices = [this.color, this.font, this.labelDisplayUnits, this.labelPrecision, this.orientation];
+}
+
+export class DotPlotSettingsModel extends Model {
+    categoryAxis = new CategoryAxisSettingsCard();
+    dataPoint = new DataPointSettingsCard();
+    labels = new LabelsSettingsCard();
+
+    cards = [this.categoryAxis, this.dataPoint, this.labels];
+
+    public setLocalizedOptions(localizationManager: ILocalizationManager) {
+        this.setLocalizedDisplayName(orientationOptions, localizationManager);
+    }
+
+    private setLocalizedDisplayName(options: IEnumMember[], localizationManager: ILocalizationManager) {
+        options.forEach(option => {
+            option.displayName = localizationManager.getDisplayName(option.displayName.toString())
+        });
+    }
+
+    public validateAndCorrectSettings(): void {
```

Comment:
Why we need to validate something? 
Doesn't it should be controlled by `minValue` and `maxValue` already?

Reply:
`NumUpDownFormat` options property which controls min and max values only checks the value of newly selected values. Meaning if in the previous version the user selected the value greater than the max value, then it will pass the validation. Here I wanted to add an additonal validation to ensure, that the radius is not an invalid value.

I think we can remove the method `validateAndCorrectSettings` as in the previous version the developer checked the value and only allowed radius to be from 1 to 15 (though in metadata the value may be between 1 and 30).

The same with `labelPrecision`. Essentially removing this additional validation breaks nothing, but the broken reports will not be fixed.

Anyway, I'll remove it 😊